readme.md

```diff
@@ -157,12 +157,36 @@ Type: `string` `Buffer` `stream.Readable` [`form-data` instance](https://github.
 
 **Note:** If you provide this option, `got.stream()` will be read-only.
 
+**Note:** If body is an object/array, it will be stringified. The `Content-Type` header will be set to `application/json` if it's not defined.
+
 The body that will be sent with a `POST` request.
 
 If present in `options` and `options.method` is not set, `options.method` will be set to `POST`.
 
 The `content-length` header will be automatically set if `body` is a `string` / `Buffer` / `fs.createReadStream` instance / [`form-data` instance](https://github.com/form-data/form-data), and `content-length` and `transfer-encoding` are not manually set in `options.headers`.
 
+###### responseType
+
+Type: `string`<br>
+Default: `text`
+
+**Note**: When using streams, this option is ignored.
+
+Parsing method used to retrieve the body from the response. Can be `text`, `json` or `buffer`. The promise has `.json()` and `.buffer()` functions which set this option automatically.
+
+Example:
+
+```js
+const {body} = await got(url).json();
+```
+
+###### resolveBody
```

Comment:
Should this be a public option?

Would there be any reason for a user to set this to `true` instead of using one of the shortcut methods?

Reply:
Custom instances:

```js
const instance = got.extend({
	resolveBody: true,
	responseType: 'json'
});

await instance(url) // returns the body
```

Reply:
> With that example there's no way to get the whole response though

Nope:

```js
instance(url, {resolveBody: false}); // returns the whole response instead
```

> So I don't think this option should be public.

Then what if someone wants the body instead of the response for all requests?

Reply:
It should be named `resolveBodyOnly`